InnerEye-DataQuality/default_paths.py

```diff
@@ -0,0 +1,15 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+
+from pathlib import Path
+
+PROJECT_ROOT_DIR = Path(__file__).parent.resolve()
+INNEREYE_DQ_DIR = PROJECT_ROOT_DIR / "InnerEyeDataQuality"
+CIFAR10_ROOT_DIR = Path("~/.torch/datasets/CIFAR10")
```

Comment:
A folder in ~ looks suspicious, where is that coming from? Our choice?

Reply:
It is the default from torchvision. Updated to put it in a subfolder of the root now.